src/algebra/module/submodule/basic.lean

```diff
@@ -194,6 +194,7 @@ p.to_sub_mul_action.smul_mem_iff' g
 instance : has_add p := ⟨λx y, ⟨x.1 + y.1, add_mem x.2 y.2⟩⟩
 instance : has_zero p := ⟨⟨0, zero_mem _⟩⟩
 instance : inhabited p := ⟨0⟩
+
```

Comment:
I'd probably drop this since this is a ported file, and style changes just add (tiny amounts of) review work when forward-porting